### PR TITLE
Migrate GH Actions installing Kubernetes to pkgs.k8s.io

### PR DESCRIPTION
**Pull request by zimnx**
Package repository we used in our CI is going to be deprecated at 13 Sep 2023.
Ref: https://kubernetes.io/blog/2023/08/31/legacy-package-repository-deprecation/

This change migrates our GH Action installing Kubernetes to use new package repository.

### CODE REVIEW

File: .github/actions/setup-kubernetes/action.yaml [hunk 1]
```diff
@@ -77,13 +77,14 @@ runs:
       set -euExo pipefail
       shopt -s inherit_errexit
       
-      export KUBERNETES_PKG_VERSION=${{ inputs.kubernetesVersion }}-00
+      export KUBERNETES_VERSION_SHORT=$( echo ${KUBERNETES_VERSION} | cut -d'.' -f1-2 )
```
Comment (rzetelskik) [inline .github/actions/setup-kubernetes/action.yaml:80]:
```suggestion
      export KUBERNETES_VERSION_SHORT=$( echo ${{ inputs.kubernetesVersion }} | cut -d'.' -f1-2 )
```
?

File: .github/actions/setup-kubernetes/action.yaml [hunk 2]
```diff
@@ -77,13 +77,14 @@ runs:
...
+      export KUBERNETES_PKG_VERSION=${{ inputs.kubernetesVersion }}-1.1
```
Comment (rzetelskik) [inline .github/actions/setup-kubernetes/action.yaml:81]: I get the change in the revision format, but why the change of the revision itself?

Comment (zimnx) [reply .github/actions/setup-kubernetes/action.yaml:81]: Looks like numbering starts from 1.1:
```
$ sudo apt-cache policy kubectl
kubectl:
  Installed: 1.25.9-00
  Candidate: 1.27.5-1.1
  Version table:
     1.27.5-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.4-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.3-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.2-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.1-1.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
     1.27.0-2.1 500
        500 https://pkgs.k8s.io/core:/stable:/v1.27/deb  Packages
